### PR TITLE
/bothelp command added 

### PR DESCRIPTION
**Pull request by ManonLef**
## Because

I would like to move forward with the final stages of the bot overhaul. This command will output information about how to see all slash commands at a glance, and maybe even more important (for learners in our discord that have been around longer); a list of which text based commands are still available, since I already see people using standalone text-based commands that have been removed. 

It would be great to also get some feedback on the remaining text-based commands, which would be:
`!xy`, `!code`, `!question`, `!google`, `!searchtop` and `!top`
If you approve of those, I can PR the complete the overhaul with removal of the remaining text based commands, old commands (like shrug, shurg, gush) and associated files. 

## This PR

- add `/bothelp` command 

Screenshot 2023-10-03 at 13 01 55

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Callbacks command: Update verbiage`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If this PR adds new features or functionality, I have added new tests
-   [ ] If applicable, I have ensured all tests related to any command files included in this PR pass, and/or all snapshots are up to date

### CODE REVIEW

**Comment by ManonLef** [comment]
@TheOdinProject/moderators tagging you as well in case you have feedback. I imagine this command will be useful to respond with in the first few days after the removal of the old commands (in case people are confused).

**Review by zachmmeyer — APPROVED** [review]
Those remaining text commands are used often, right?

**Comment by ManonLef** [comment]
@zachmmeyer they're the ones most often used in a sentence like;

> Please ask a !question including your !code

> This sounds like an !xy problem to me

> I'd suggest googling for this problem !google mdn Fetch API 

etc. 

If you feel some could be removed or should be added, let me know. I was doubtful of !xy but when searching the discord, I still saw it used in the help channels quite a bit.

**Comment by Mclilzee** [comment]
With the incoming changes to /help command why are we creating another help one like this? wouldn't it be better if we make it a sub command of the /help? `/help bot` for example?

**Comment by ManonLef** [comment]
@zachmmeyer what do you mean by porting them all over? Most commands are converted to slash now. These are as well.

**Comment by zachmmeyer** [comment]
I misunderstood. I'm good with leaving the mentioned text-based commands.
I'm also in favor of letting some time to go by to allow anyone else on the team to share their opinion.
Also in favor of moving ahead.

**Comment by ManonLef** [comment]
I agree and would love more input. I know both @01zulfi and @wise-king-sullyman were involved in other PR's addressing the switch. Also in regards to enhance the slash command workflow, testing etc. Maybe you both have some input. 

I am hoping the removal of the old stuff will remove clutter and hopefully makes it easier to progress with organising the repo towards better scalability

**Comment by Baguirre03** [comment]
I personally use : `!code`, `!question`, `!google`, and `!top` _often_ in sentences. Would hate to see them go. The other two less often.

**Comment by JustWaveThings** [comment]
> !xy, !code, !question, !google, !searchtop and !top

I'd keep !code, !question, and !top text commands. I think the other text commands are better as slash commands.

File: new-era-commands/slash/bothelp.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      .setDescription(`
+**Slash Commands** 
+Type a slash \`/\` to see a list of all the available slash commands you can use. Most commands offer extra options, such as pinging a user or specifying additional variables.
+
+**Text-Based Commands** 
+We also have these text-based commands which you can use in a sentence:      
+\`!xy\`, \`!code\`, \`!question\`, \`!google\`, \`!searchtop\` and \`!top\`
+
+**Bot Playground** 
+Experiment with our bot in the [dedicated bot channel](https://discord.com/channels/505093832157691914/513125912070455296) to try out different commands and features. It's a safe space to explore without disturbing the main channels.
```
Comment (01zulfi) [inline new-era-commands/slash/bothelp.js:23]: We can directly mention the channel with 

```
<#123456>
```

Where 123456 is the channel ID. You can get bot-spam-playground's channel ID in the `config.js` file

**Review by 01zulfi — APPROVED** [review]
lgtm 🚀 

Just one small nit below

File: new-era-commands/slash/bothelp.js (hunk 1 above)
Comment (ManonLef) [reply new-era-commands/slash/bothelp.js:23]: sweet! Thank you

**Comment by ManonLef** [comment]
Screenshot 2023-10-07 at 22 24 40

updated after feedback from @01zulfi above

**Review by 01zulfi — APPROVED** [review]

**Pull request merged**